packages/starlight/utils/user-config.ts

```diff
@@ -125,6 +125,8 @@ const UserConfigSchema = z.object({
       github: z.string().url().optional(),
       /** Link to the Discord server for this site, e.g. `'https://astro.build/chat'`. */
       discord: z.string().url().optional(),
+      /** Link to the Codeberg server for this site, e.g. `'https://codeberg.org/knut/examples'`. */
```

Comment:
I guess “server” is not quite the right word in the case of Codeberg. Maybe we can say “profile or repository”?

```suggestion
      /** Link to the Codeberg profile or repository for this site, e.g. `'https://codeberg.org/knut/examples'`. */
```